src/components/utils/OffchainUtils.ts

```diff
@@ -3,7 +3,8 @@ import axios from 'axios'
 import config from 'src/config'
 import { AccountId, ElasticQueryParams } from 'src/types'
 
-const { offchainUrl, subIdApiUrl } = config
+let { offchainUrl, subIdApiUrl } = config
+subIdApiUrl = 'http://localhost:3001/api/v1'
```

Comment:
you have sub.id URL in connection config

<img width="424" alt="image" src="https://github.com/dappforce/polkaverse/assets/39906111/8e524cba-c93d-4077-a74e-482c32aa0857">

Reply:
yeah that was wrong, I changed it for local development, forgot to change it back